pkg/test/util/file/file.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package file
+
+import (
+	"io/ioutil"
+	"testing"
+)
+
+// AsBytes is a utility function that reads the content of the given file
+// and fails the test if an error occurs.
+func AsBytes(t testing.TB, filename string) []byte {
```

Comment:
Consider following the naming convention in this PR (e.g., DeleteSecretOrFail) to name this function as AsBytesOrFail.

Reply:
I had considered that but decided against it for two reasons:

1. It's overly verbose from the call sites. Most of the time I found that it was being called within the context of another `XXXOrFail` call, leading to `xxxOrFail(...AsBytesOrFail(...))`, which seemed a bit ugly.
2. There is no need for a non-`OrFail` version (since we have `ioutil.ReadFile`), so there's no need to distinguish this version of the method.